Foundation/NSPropertyList.swift

```diff
@@ -113,6 +113,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return data
     } else if let number = input as? NSNumber {
         return number
+    } else if let url = input as? NSURL {
```

Comment:
Can you track down where that is? I would be surprised if we inserted an NSURL there; since as @phausler says it is not a plist type.